nifi-registry/pom.xml

```diff
@@ -35,6 +35,10 @@
         <module>nifi-registry-docker-maven</module>
     </modules>
     <properties>
+        <nifi.registry.jetty.version>12.0.25</nifi.registry.jetty.version>
```

Comment:
Is there a reason to introduce this new property as opposed to overriding the value from the parent Maven configuration?